src/main/scala/org/apache/mesos/chronos/scheduler/jobs/stats/JobStats.scala

```diff
@@ -190,7 +193,9 @@ class JobStats @Inject()(clusterBuilder: Option[Cluster.Builder], config: Cassan
    * @return updated TaskStat object
    */
   private def updateTaskStat(taskStat: TaskStat, row: Row): TaskStat = {
-    val taskTimestamp = row.getDate(TIMESTAMP)
+    val localDate = row.getDate(TIMESTAMP)
+    calendar.set(localDate.getYear, localDate.getMonth, localDate.getDay)
+    val taskTimestamp = calendar.getTime
```

Comment:
Alright :relieved: Sorry, I'm easily panicked by changes to time stuff :sweat_smile: